### ISSUE TITLE
is this library still being maintained?

### ISSUE THREAD

**Issue by nonoumasy** [jloosli/node-firestore-import-export#866]

**Comment by cuong0993** [comment]
Forked here https://github.com/cuong0993/node-firestore-import-export

**Comment by nonoumasy** [comment]
Thanks! Does this resolve the "Bandwidth Exhausted" issue?
https://github.com/jloosli/node-firestore-import-export/issues/411

**Comment by arthabus** [comment]
@cuong0993 could you please have a look at this one #869?

Maybe you have a clue what needs to be updated as I'm not that familiar with the code. After that I could probably fix it myself.

**Comment by cuong0993** [comment]
> Thanks! Does this resolve the "Bandwidth Exhausted" issue? #411

No. I believe the "Bandwidth Exhausted" issue is from the server.
It might occur with a large database import/export.
This lib is simple and do not handle that kind of error, unfortunately.
Refer https://github.com/googleapis/nodejs-datastore/issues/525

**Comment by arthabus** [comment]
@nonoumasy @cuong0993 just run across the below code that deals with "Bandwidth Exhausted"

https://gist.github.com/navidshad/973e9c594a63838d1ebb8f2c2495cf87

maybe either of you guys have capacity to integrate it into the current code?
